core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -59,6 +59,7 @@ abstract class SparkFunSuite
   protected val enableAutoThreadAudit = true
 
   protected override def beforeAll(): Unit = {
+    System.setProperty("spark.testing", "true")
```

Comment:
why we need this?

Reply:
If this parameter is not set, sys.props.contains ("spark.testing") is false.

Reply:
Sorry let me make the question more clear.

Why we need this if `./project/SparkBuild.scala:795: javaOptions in Test += "-Dspark.testing=1"` works?

Reply:
My debugging tool is IDEA, I think IDE had no relevance to the process of setting.
Be similar to HiveSparkSubmitSuite RPackageUtilsSuite SparkSubmitSuite.
There are also manually add System.setProperty("spark.testing", "true").
Of course， I try with mavn(using command line) to test case , it is right.  thanks.

Reply:
if we are already doing this, let's make it more explicit that we should remove `./project/SparkBuild.scala:795: javaOptions in Test += "-Dspark.testing=1"` and set `spark.testing` in `SparkFunSuite.beforeAll`.

Reply:
@cloud-fan ,thanks, I have update it. Can you help me to test it. thanks.